### PR TITLE
try building v0.11.0

### PR DESCRIPTION
**Pull request by grlee77**
numpy requirement was bumped to 1.10 in this release

### CODE REVIEW

**Comment by grlee77** [comment]
@hgomersall and @jakirkham :
I thought I would see if all is well with the current v0.11.0 tag here before officially creating the release on GitHub.

probably want to add 3.7 now as well. not sure we should drop 3.5 quite yet. thoughts?

**Comment by hgomersall** [comment]
I agree re 3.7 and not dropping 3.5.

**Comment by grlee77** [comment]
All cases were green except for the OS X + Python 3.7 which fails immediately for some reason.

All OS X builds are giving a warning about `xcode6.4` being depricated with support to be removed in Jan2019.

**Comment by grlee77** [comment]
Hi @cancan101
Thanks for setting up pyFFTW-wheels! we are currently in the process of making a new release. I bumped the multibuild submodule and have been attempting to add Python 3.7. Let me know if you have any feedback on the PR.

**Comment by cancan101** [comment]
lgtm

**Comment by grlee77** [comment]
I don't know why that one Python 3.7 case is trying to download Python 3.5 and failing, but the others are not. They are all set to `language: generic` so I don't see what the difference is.

https://travis-ci.org/pyFFTW/pyFFTW-wheels/jobs/452479159#L26-L29

**Comment by grlee77** [comment]
The last commit here refactors the `.travis.yaml` slightly and seems to have fixed the previous issue. I still don't know why only the 3.7 test case had an issue with the prior settings, though. 

I assume we had xcode6.4 as the oldest available so that we can be backward compatible with older OS X 10.10?. For that reason, I have left that as-is for now. According to the [Travis CI docs](https://docs.travis-ci.com/user/languages/objective-c/#supported-xcode-versions), we can bump to xcode7.3 or 8.0 and build using OS X 10.11 instead of 10.10 once support for xcode6.4 has been dropped.

Can someone please review/approve so we can merge? Thanks

File: .travis.yml [hunk 1]
```diff
@@ -57,14 +66,18 @@ matrix:
...
-        - NP_BUILD_DEP=numpy==1.9.3
-        - NP_TEST_DEP=numpy==1.9.3
     - os: osx
       language: generic
       env:
         - MB_PYTHON_VERSION=3.6
         - NP_BUILD_DEP=numpy==1.11.3
         - NP_TEST_DEP=numpy==1.11.3
+    - os: osx
+      langugage: objective-c
```
Comment (jakirkham) [inline .travis.yml]: Interesting. Why does it need `objective-c` here and use `generic` in the others?

Comment (grlee77) [reply .travis.yml]: I don't think it does. I had meant to change that back to generic.

Comment (grlee77) [reply .travis.yml]: since `language: generic` is at the top now, I guess we don't need to list anything for language on these OS X cases.

**Comment by grlee77** [comment]
@jakirkham: Hopefully this is good to go now. I removed the stray `objective-c` and things are still all green.

**Comment by grlee77** [comment]
@hgomersall: Once this PR is merged, the OS X and Linux wheels for 0.11 should get built and show up at:
https://3f23b170c54c2533c070-1c8a9b3114517dc5fe17b7c3f8c63a43.ssl.cf2.rackcdn.com/

From there you can download them and upload to PyPI manually (or via use of the "wheel-uploader" script described in this repo's README.rst).

I would also be happy to do the uploading, but would need you to issue the necessary permissions on PyPI first.

For windows wheels, I started a PR in #24 to work on building them from this repo, but it will need you to configure Appveyor to connect with this repo to proceed. It looks like previously, wheels were automatically uploaded to your bintray account, but that functionality was broken at the time I tagged 0.11 so they did not appear there. It might be fixed by the currently open pyfftw/pyfftw#248, but I wasn't clear on whether merging that would cause them to appear. If they do, I am concerned that versioneer would cause a modified version tag as master would no longer exactly corresponding to the v0.11.0 tag after merging the new PR.  Using this repo to build the windows wheels would be building from the v0.11.0 tag, so I am not worried about it getting the version wrong if we go with that route.

File: .travis.yml [hunk 2]
```diff
@@ -19,52 +19,61 @@ env:
...
 matrix:
-  exclude:
-    # Exclude the default Python 3.5 build
-    - python: 3.5
   include:
     - os: linux
+      language: python
+      # The travis Python version is unrelated to the version we build and test
+      # with.  This is set with the MB_PYTHON_VERSION variable.
+      python: 3.5
```
Comment (jakirkham) [inline .travis.yml]: Sorry for the additional question, but are these needed? Just wondering given the comment here. Though I'm probably just clueless about how this all works. :)

Comment (grlee77) [reply .travis.yml]: I'm not really sure, I was just trying to keep the linux behavior as it was previously. I will just try removing it and see what happens.

Comment (grlee77) [reply .travis.yml]: From a quick look at the files in the `multibuild` submodule, it seems to be assumed that python is present on the system so I think some version of python needs to be installed, but it shouldn't matter which one. Within the build scripts, a `manylinux1` docker image is downloaded and the python version under `MB_PYTHON_VERSION` gets passed to the docker `run` command.

Comment (grlee77) [reply .travis.yml]: looks like everything does still pass without those lines

File: .travis.yml [hunk 3]
```diff
@@ -42,29 +36,32 @@ matrix:
     - os: linux
       env:
         - MB_PYTHON_VERSION=3.5
-        - NP_BUILD_DEP=numpy==1.9.3
-        - NP_TEST_DEP=numpy==1.9.3
```
Comment (jakirkham) [inline .travis.yml:46]: Shouldn't these still  be here?

File: .travis.yml [hunk 4]
```diff
@@ -42,29 +36,32 @@ matrix:
...
     - os: osx
-      language: generic
       env:
         - MB_PYTHON_VERSION=2.7
     - os: osx
-      language: generic
       env:
         - MB_PYTHON_VERSION=3.5
-        - NP_BUILD_DEP=numpy==1.9.3
-        - NP_TEST_DEP=numpy==1.9.3
```
Comment (jakirkham) [inline .travis.yml:61]: Same as above.

File: .travis.yml (hunk 3 above)
Comment (jakirkham) [reply .travis.yml:46]: NVM. They are set to 1.10 globally. Sorry for the noise.

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
LGTM. @cancan101, would you be able to review?

**Comment by cancan101** [comment]
lgtm

**Comment by hgomersall** [comment]
lgtm

**Comment by grlee77** [comment]
It think this is ready to go.
I am apparently not a member of the "CI" team under the pyFFTW org, so one of you will have to merge this.

**Pull request merged**